src/WebJobs.Script.WebHost/Security/DefaultKeyValueConverterFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public sealed class DefaultKeyValueConverterFactory : IKeyValueConverterFactory
+    {
+        private static readonly PlaintextKeyValueConverter PlainTextConverter = new PlaintextKeyValueConverter();
+
+        public IKeyValueConverter GetValueConverter(Key key, KeyConversionAction action)
```

Comment:
You could also just use System.IO.FileAccess rather than define a new enum. We do that elsewhere already. It seems like it opens the door to confusion to have a hint like this that isn't enforced. I could specify Read, but do read/writes.